pkg/shell/shell.go

```diff
@@ -46,8 +46,14 @@ func setupShell(fds [3]*os.File, p Paths, spawn bool) (*eval.Evaler, func()) {
 	ev := InitRuntime(fds[2], p, spawn)
 	restoreSHLVL := incSHLVL()
 
+	// This implicitly catches every signal regardless of whether it is
+	// currently ignored.
 	sigs := make(chan os.Signal)
 	signal.Notify(sigs)
+	// TODO: Remove this if, and when, job control is implemented. This must
```

Comment:
Document that this `Ignore` call is purely so that external commands can inherit the ignore flag.

Reply:
Done. I added another sentence to that effect even though I thought this was clear from the first sentence after the "TODO:" prefix.